### ISSUE TITLE
Performance is worse while training

### ISSUE THREAD

**Issue by 186big** [mit-han-lab/bevfusion#582]
Thank u for your great job.
I use your code and pretrained model to train a Lidar + Camera detection model in Nuscene dataset , but the more epochs I trained, the worse it  got. 
The loss is lower But the map is lower too.
![image](https://github.com/mit-han-lab/bevfusion/assets/97373109/bae3450f-1a9d-4eab-80d6-e2ba2b1d3564)

**Comment by 971022jing** [comment]
> Thank u for your great job. I use your code and pretrained model to train a Lidar + Camera detection model in Nuscene dataset , but the more epochs I trained, the worse it got. The loss is lower But the map is lower too. ![image](https://private-user-images.githubusercontent.com/97373109/293881983-bae3450f-1a9d-4eab-80d6-e2ba2b1d3564.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ0MjI0MDQsIm5iZiI6MTcwNDQyMjEwNCwicGF0aCI6Ii85NzM3MzEwOS8yOTM4ODE5ODMtYmFlMzQ1MGYtMWE5ZC00ZWFiLTgwZDYtZTJiYTJiMWQzNTY0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA1VDAyMzUwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM0YzZkNzY1YTk4ZDVjZGM2NDA4OGRkZDFlZjEzN2ExMGYxNTg1YzM1NjY1ZWE3MmI5ZGQ0MzEwZjY1YzQ1MmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2sjRIOVIo8puXQn0a1pIZZCzgWbCqDtUcQo_DVLsubs)
hello，have you solved the problem?

**Comment by 186big** [comment]
> > Thank u for your great job. I use your code and pretrained model to train a Lidar + Camera detection model in Nuscene dataset , but the more epochs I trained, the worse it got. The loss is lower But the map is lower too. ![image](https://private-user-images.githubusercontent.com/97373109/293881983-bae3450f-1a9d-4eab-80d6-e2ba2b1d3564.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ0MjI0MDQsIm5iZiI6MTcwNDQyMjEwNCwicGF0aCI6Ii85NzM3MzEwOS8yOTM4ODE5ODMtYmFlMzQ1MGYtMWE5ZC00ZWFiLTgwZDYtZTJiYTJiMWQzNTY0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA1VDAyMzUwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM0YzZkNzY1YTk4ZDVjZGM2NDA4OGRkZDFlZjEzN2ExMGYxNTg1YzM1NjY1ZWE3MmI5ZGQ0MzEwZjY1YzQ1MmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2sjRIOVIo8puXQn0a1pIZZCzgWbCqDtUcQo_DVLsubs)
> > hello，have you solved the problem?

sorry, not yet

**Comment by Li-Whasaka** [comment]
using full dataset?

**Comment by 186big** [comment]
> using full dataset?

Yes

**Comment by Acc2386** [comment]
same problem, have you salved this?

**Comment by zhijian-liu** [comment]
Thank you for your interest in our project. This repository is no longer actively maintained, so we will be closing this issue. Please refer to the amazing implementation at [MMDetection3D](https://github.com/open-mmlab/mmdetection3d/tree/main/projects/BEVFusion). Thank you again!
